app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php

```diff
@@ -139,7 +139,7 @@ public static function parseList($list, $removeEmpty = true, $ignoreNumerical =
         $choices = [];
 
         if ($list) {
-            foreach ($list as $val => $label) {
+            foreach ((array) $list as $val => $label) {
```

Comment:
Did you hit a notice with this sparking this change and is it reproducible? If so, worries me we have a bug elsewhere and this is just a patch.